test/testcafe/spec/Identify_spec.js

```diff
@@ -317,3 +317,14 @@ test.requestHooks(identifyRequestLogger, baseIdentifyMock)('should pre-populate
   const identifier = identityPage.getIdentifierValue();
   await t.expect(identifier).eql('myTestUsername@okta.com');
 });
+
+test.requestHooks(identifyRequestLogger, baseIdentifyMock)('should hide "Keep me signed in" checkbox with config', async t => {
+  const identityPage = await setup(t);
+  await rerenderWidget({
+    features: { showOptionForKeepMeSignedIn: false }
+  });
+
+  // Ensure checkbox is hidden
+  const doesCheckboxExist = identityPage.identifierFieldExists('.custom-checkbox [name="rememberMe"');
+  await t.expect(doesCheckboxExist).eql(false);
+});
```

Comment:
Is there a test for when this feature is true?

Reply:
Added a test case for this and the default scenario